packages/runtime-logger/tsup.config.ts

```diff
@@ -0,0 +1,19 @@
+import { defineConfig } from 'tsup';
+
+export default defineConfig({
+	entry: ['src/index.ts'],
+	format: ['esm'],
+	target: 'node18',
+	bundle: true,
+	dts: {
+		banner: '/// <reference path="../virtual.d.ts" />\n',
```

Comment:
I'm curious, why is this needed?

Reply:
With this the types from `virtual.d.ts` are loaded just by importing this library. You don't need to add a reference yourself.

Reply:
But aren't you importing `virtual.d.ts` already?

Reply:
my bad i thought you were importing it